src/vs/editor/contrib/find/browser/findOptionsWidget.css

```diff
@@ -0,0 +1,12 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+
+.monaco-editor .findOptionsWidget {
+	color: var(--vscode-editorWidget-foreground);
+	background-color: var(--vscode-editorWidget-background);
+	box-shadow: var(--vscode-widget-shadow);
```

Comment:
That should be `0 0 8px 2px var(--vscode-widget-shadow)`

Reply:
oh! Thanks😆